non-critical-infra/packages/encrypt-email-address/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  python3,
+  ruff,
+  sops,
+}:
+
+python3.pkgs.buildPythonApplication {
+  name = "encrypt-email-address";
+  src = ./.;
+
+  format = "other";
+
+  nativeBuildInputs = [ ruff ];
+
+  propagatedBuildInputs = [ python3.pkgs.click ];
+
+  installPhase = ''
+    ruff check ./encrypt-email-address.py
```

Comment:
Do we really want this to potentially fail to build when ruff is being updated?

Reply:
Yes, we control the nixpkgs instance and the source here, so we should address new errors. However we already do this check in https://github.com/NixOS/infra/blob/master/formatter/flake-module.nix so this seem a bit redundant.

```suggestion
```

Reply:
> However we already do this check in https://github.com/NixOS/infra/blob/master/formatter/flake-module.nix so this seem a bit redundant

That's `ruff-format`, not `ruff-check`. But good point! I've added that over in https://github.com/NixOS/infra/pull/498

Reply:
I've removed the call to `ruff check` (in favor of #498). I'm marking this resolved.